backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/handlers/DeleteFromDiagramEventHandler.java

```diff
@@ -176,6 +180,8 @@ private IStatus invokeDeleteNodeTool(Node node, IEditingContext editingContext,
                 variableManager.put(VariableManager.SELF, optionalSelf.get());
                 variableManager.put(IDiagramContext.DIAGRAM_CONTEXT, diagramContext);
                 variableManager.put(Node.SELECTED_NODE, node);
+                variableManager.put(ELEMENT_VIEW, node);
+                variableManager.put(CONTAINER_VIEW, this.getParentNode(node, diagramContext.getDiagram()));
```

Comment:
This code belong in the compatibility layer, not in the code. We provide the variable selected node, no need to provide it here under a different name.

Reply:
Done